.github/workflows/helmtagbumper.yaml

```diff
@@ -53,18 +54,6 @@ jobs:
           GITHUB_REPO: datarepo-helm-definitions
           SWITCH_DIRECTORIES: true
           MERGE_BRANCH: master
-      - name: "Notify Slack"
-        if: always()
-        uses: broadinstitute/action-slack@v3.8.0
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
-        with:
-          status: ${{ job.status }}
-          fields: job,repo,message,author,took
-          author_name: "[API] [datarepo-helm-definitions] Version update for Integration namespaces"
-          mention: fb,muscles
-          mention_if: failure
```

Comment:
Note to reviewers: Since we're using a reusable workflow, I have consolidated it so that we have one slack notification after all of the helm tasks run, instead of three.

Reply:
Update: I decided to leave this step in place, but have it only trigger when the action is triggered via workflow_**dispatch**. Normally, it will be triggered via workflow_**call**. So, I think it would be good to still get a notification if it's run out of cycle.